docker/CMSRucioClient/scripts/k8s_sync_users_links.sh

```diff
@@ -33,3 +33,20 @@ fi
 
 echo "Creating links"
 ./cmslinks.py --overwrite --disable
+
+# Preparing oidc-agent to get/refresh token to retrieve OIDC user information
+export OIDC_CONFIG_DIR=$HOME/.oidc-agent
```

Comment:
How does .oidc-agent get created/filled?

Reply:
if it does not exists, the oidc-agent will create it.

Reply:
basically to store configuration for the token renewal